barman/cli.py

```diff
@@ -2103,6 +2148,111 @@ def manage_server_command(
     return True
 
 
+def get_model_list(args=None):
```

Comment:
I'm REALLY nitpicking here, but this should be get_models_list I guess? plural...

Reply:
I think so too, I only did it that way to follow the pattern of server related functions -- `get_server`, `get_server_list`, and so on 😆

Reply:
(Not to mention it returns a mapping, not a list heh)